API/Controllers/ProjectController.cs

```diff
@@ -239,6 +239,35 @@ public async Task<IActionResult> MigrateDatabaseToElasticSearch()
             return Ok(resultsResource);
         }
 
+        /// <summary>
+        ///     This method returns suggestions while searching for projects
+        /// </summary>
+        /// <param name="query"></param>
+        /// <returns>This method returns a list of autocomplete project resources.</returns>
+        /// <response code="200">This endpoint returns a list with suggested projects.</response>
+        [HttpGet("search/autocomplete")]
+        [ProducesResponseType(typeof(List<AutocompleteProjectResourceResult>), (int) HttpStatusCode.OK)]
+        public async Task<IActionResult> GetAutoCompleteProjects([FromQuery(Name ="query")] string query)
```

Comment:
When testing I noticed the searching only works for queries that are typed from the beginning of a word.
_For example:
Searching for the word: "Spe", might return a project called: "This Special Delivery".
However, searching for: "pecial" won't give result._
Is this intentional? a bug? or a limitation of the system?

Reply:
Looking through the code I saw this was intentional based on the summary of the method: 'FindProjectsWhereTitleStartsWithQuery'.

Query's description: _The string with which the title must begin_

(Might be nice to convey this to users in the frontend in some way)

Reply:
Agree, should have been more specific on the steps to test. 

This is intentional but maybe it the future we should also make you're suggestion possible.